src/app/blog/components/post-detail/post-detail.component.ts

```diff
@@ -30,7 +30,7 @@ export class PostDetailComponent implements OnInit {
   // Pattern to valid email adress
   private emailPattern = '^[a-z0-9._%+-]+@[a-z0-9.-]+\\.[a-z]{2,4}$';
 
-  // @Irice why should i use ActivatedRoute  in this Component?
+  // @Irice why should i use ActivatedRoute  in this Component? @Ghislain this is load the information containing on the post DataBase
```

Comment:
**Not correct: comment **
Read this: https://angular.io/api/router/ActivatedRoute

Reply:
ActivatedRoute   is ANGULAR BASIC COncept : you should know that even if you sleep

Reply:
ok